test/unit/foreman/renderer/snapshots/ProvisioningTemplate/provision/Kickstart default.snap.txt

```diff
@@ -41,22 +41,11 @@ wget
 redhat-lsb-core
 %end
 
-%post --nochroot
-exec < /dev/tty3 > /dev/tty3
-#changing to VT 3 so that we can see whats going on....
-/usr/bin/chvt 3
-(
-cp -va /etc/resolv.conf /mnt/sysimage/etc/resolv.conf
-/usr/bin/chvt 1
-) 2>&1 | tee /mnt/sysimage/root/install.postnochroot.log
+%post --nochroot --log=/mnt/sysimage/root/install.postnochroot.log
 %end
 
 %post --log=/root/install.post.log
 logger "Starting anaconda snapshothost postinstall"
-exec < /dev/tty3 > /dev/tty3
-#changing to VT 3 so that we can see whats going on....
-/usr/bin/chvt 3
```

Comment:
I always found this to be useful to see what's going on. Especially on RHEL where pretty much everything is installed in `%post`, unlike CentOS where you actually have a progress indication. Please keep this in.

Reply:
It does not work in EL8 where tmux is used.